### ISSUE TITLE
The path of the image in type.md under src/python/docs/sphinx is not correct

### ISSUE THREAD

**Issue by Stephen0620** [microsoft/NimbusML#148]
After running make_yaml.bat under src/python/docs/sphinx, the path of loading the image in type.md is not correct and needs to be adjusted manually 

**To Reproduce**
1. Go to 'src/python/docs/sphinx'
2. run make_yaml.bat
3. Check line 67, the path is showing ../_images/concepts\../_static/images/table_car.png

**Expected behavior**
It should be ../_images/_static/images/table_car.png

**Comment by ganik** [comment]
is this fixed by #144 ?

**Comment by Stephen0620** [comment]
> is this fixed by #144 ?

No, not yet, there is still a few issues while generating the documentation and needs to be adjusted manually
